packages/material-ui-based-components/src/components/Title.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { withRuleEngine, richTextString } from '../shared/withRuleEngine';
+import { PROPS } from '../utils/types';
+import { Typography } from '@mui/material';
+
+
+const TitleComponent = (props: PROPS) => {
+    const { text } = props;
+
+    return (
+        <Typography gutterBottom>{richTextString(text)}</Typography>
```

Comment:
please use richText property for rendering rich text or not

Reply:
added richText property